examples/tts/conf/rad-tts_feature_pred.yaml

```diff
@@ -0,0 +1,339 @@
+        use_context_lstm: true
+        context_lstm_norm: spectral
+        context_lstm_w_f0_and_energy: true
+        text_encoder_lstm_norm: spectral
+        use_text_conditional_priors: false
+        zero_out_context: false
+        n_aug_dims: 6
+        n_f0_dims: 1
+        n_energy_avg_dims: 1
+        use_first_order_features: false
+        unvoiced_bias_activation: "relu"
+        decoder_use_partial_padding: false
+        decoder_use_unvoiced_bias: true
+        ap_pred_log_f0: true
+        ap_use_unvoiced_bias: true
+        ap_use_voiced_embeddings: true
+        p_dropout: 0.1
+        noise_to_unvoiced_in_f0: 0
+        noise_to_pvoiced: 0
+        dur_model_config: 
+            name: dap 
+            hparams: 
+                #_target_: "nemo.collections.tts.modules.feature_prediction_model.FP"
+                n_speaker_dim: 16 
+                bottleneck_hparams: 
+                    in_dim: 512
+                    reduction_factor: 16
+                    norm: weightnorm
+                    non_linearity: relu
+                take_log_of_input: true
+                arch_hparams: 
+                    out_dim: 1
+                    n_layers: 2
+                    n_channels: 256
+                    kernel_size: 3
+                    p_dropout: 0.1
+        f0_model_config: 
+            name: dap
+            hparams:
+                n_speaker_dim: 16
+                bottleneck_hparams: 
+                    in_dim: 512
+                    reduction_factor: 16
+                    norm: weightnorm
+                    non_linearity: relu
+                take_log_of_input: false
+                arch_hparams: 
+                    out_dim: 1
+                    n_layers: 2
+                    n_channels: 256
+                    kernel_size: 11
+                    p_dropout: 0.5
+
+        energy_model_config: 
+            name: dap
+            hparams:
+                n_speaker_dim: 16
+                bottleneck_hparams: 
+                    in_dim: 512
+                    reduction_factor: 16
+                    norm: weightnorm
+                    non_linearity: relu
+                take_log_of_input: false
+                arch_hparams: 
+                    out_dim: 1
+                    n_layers: 2
+                    n_channels: 256
+                    kernel_size: 3
+                    p_dropout: 0.5
+        v_model_config : 
+            name: dap
+            hparams:
+                n_speaker_dim: 16
+                take_log_of_input: false
+                bottleneck_hparams: 
+                    in_dim: 512
+                    reduction_factor: 16
+                    norm: weightnorm
+                    non_linearity: relu
+                arch_hparams: 
+                    out_dim: 1
+                    n_layers: 2
+                    n_channels: 256
+                    kernel_size: 3
+                    p_dropout: 0.5
+
+
+
+        
+
+trainer:
+  gpus: 8
```

Comment:
Should this be replaced with `devices` and `accelerator` params?

Reply:
changed to devices and accelerator. Nice catch